tests/helpers/utils.ts

```diff
@@ -136,7 +141,7 @@ export async function createAddon({
     console.debug(`\tember ${emberCliArgs.join(' ')}`);
   }
 
-  let result = await execa('ember', emberCliArgs, {
+  let result = await execa('npx', ['ember-cli@beta', ...emberCliArgs], {
```

Comment:
I know we're in beta mode but I'm really uncomfortable doing this in our blueprint.

We can put it as latest if you like and work to backport if we think it's important but I really don't think we can use beta here

Reply:
We can't get ci passing otherwise.

We'd just have to wait until 5.2 is released to do anything

Reply:
sounds like it needs to be backported then. Also it's release week 🤷